pkg/common/startrpc/start.go

```diff
@@ -61,16 +65,15 @@ func Start(
 	if err != nil {
 		return err
 	}
+	var reg *prometheus.Registry
+	var metric *grpcprometheus.ServerMetrics
 	// ctx 中间件
 	if config.Config.Prometheus.Enable {
-		prome.NewGrpcRequestCounter()
-		prome.NewGrpcRequestFailedCounter()
-		prome.NewGrpcRequestSuccessCounter()
-		unaryInterceptor := mw.InterceptChain(grpcprometheus.UnaryServerInterceptor, mw.RpcServerInterceptor)
-		options = append(options, []grpc.ServerOption{
-			grpc.StreamInterceptor(grpcprometheus.StreamServerInterceptor),
-			grpc.UnaryInterceptor(unaryInterceptor),
-		}...)
+		//////////////////////////
+		cusMetrics := prom_metrics.GetGrpcCusMetrics(rpcRegisterName)
+		reg, metric, err = prom_metrics.NewGrpcPromObj(cusMetrics)
```

Comment:
```suggestion
		reg, metric, err = prom_metrics.NewGrpcPromObj(cusMetrics)
```
It appears that the errors here have not been handled properly. May I suggest taking a closer look at the code and implementing some improvements to ensure that errors are handled more effectively in the future? One potential solution could be to use a try-catch block to handle any potential errors that may arise.